.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: main
+on: push
+jobs:
+  node_18:
```

Comment:
You don't need to duplicate the jobs here. You can use a build matrix. See https://github.com/CesiumGS/obj2gltf/blob/main/.github/workflows/build.yml for a simple example.